packages/mesos-metrics-module/build

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir -p build
+pushd build
+
+# configure module build with cmake
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DTESTS_ENABLED=False \
+  -DCMAKE_INSTALL_PREFIX=$PKG_PATH \
+  -Dmesos_INCLUDE_DIR=/opt/mesosphere/include \
+  -Ddefault_BIN_DIR=/opt/mesosphere/bin \
+  -Ddefault_LIB_DIR=/opt/mesosphere/lib \
+  -DUSE_LOCAL_PROTOBUF=False \
+  -Dprotobuf_INCLUDE_DIR=/opt/mesosphere/lib/mesos/3rdparty/include \
```

Comment:
There's currently `HINTS` entries which default to `${mesos_INCLUDE_DIR}`. Could add a second HINT which points to `${default_LIB_DIR}/mesos/3rdparty/include`:
- glog: https://github.com/dcos/dcos-metrics/blob/master/mesos_module/CMakeLists.txt#L148
- protobuf: https://github.com/dcos/dcos-metrics/blob/master/mesos_module/CMakeLists.txt#L124

see also: https://cmake.org/cmake/help/v3.0/command/find_path.html